site-src/geps/overview.md

```diff
@@ -0,0 +1,119 @@
+# Gateway Enhancement Proposal (GEP)
+
+Gateway Enhancement Proposals (GEPs) serve a similar purpose to the [KEP][kep]
+process for the main Kubernetes project:
+
+1. Ensure that changes to the API follow a known process and discussion
+  in the OSS community.
+1. Make changes and proposals discoverable (current and future).
+1. Document design ideas, tradeoffs, decisions that were made for
+  historical reference.
+
+## Process
+
+### 1. Circulate the Idea
+Before creating a GEP, share your high level idea with the community. This can
+be in one of many forms:
+
+- A [new GitHub Discussion](https://github.com/kubernetes-sigs/gateway-api/discussions/new)
+- On our [Slack Channel](https://kubernetes.slack.com/archives/CR0H13KGA)
+- On one of our [community meetings](contributing/community/#meetings)
+
+### 2. Agree on the Goals
+Although it can be tempting to start writing out all the details of your
+proposal, it's important to first ensure we all agree on the goals. The first
+version of your GEP should aim for a "Provisional" status and leave out any
+implementation details, focusing primarily on "Goals" and "Non-Goals".
```

Comment:
:+1:

Reply:
👍 Makes sense to codify this, as this has been encouraged for several recent GEPs, and jumping directly to implementation has definitely been a stumbling block.